src/github.com/getlantern/balancer/balancer.go

```diff
@@ -71,13 +71,19 @@ func (b *Balancer) DialQOS(network, addr string, targetQOS int) (net.Conn, error
 
 	_, port, _ := net.SplitHostPort(addr)
 
-	if port == "" || port == "80" || port == "8080" {
+	// try proxy non-https traffic through trusted servers first
+	if port != "443" {
```

Comment:
There's an issue with this change and it's that we support many other different ports:

```
./flashlight.go:                AllowedPorts: []int{80, 443, 8080, 8443, 5222, 5223, 5228},
```

From that list we only want to take ports `80` an `8080` as probably untrusted. It is true that plaintext traffic may travel along any port but I think we are not ready to make that movement and claim that Lantern routes all plaintext traffic to trusted servers, as of today it would be more accurate to say we proxy traffic that passes over ports `80` and `8080` to trusted servers and that users must take care of other ports.